landingzones/tests/test_permissions.py

```diff
@@ -199,313 +206,3 @@ def test_zone_create_disable(self):
         ]
         self.assert_response(url, good_users, 200)
         self.assert_response(url, bad_users, 302)
-
-    def test_zone_update(self):
```

Comment:
Unless I'm mistaken, it should be possible to keep the zone update and delete permission tsts in `test_permissions`, as no taskflow creation is needed to access these views.